plugins/catalog-unprocessed-entities/src/components/PendingEntities.tsx

```diff
@@ -57,22 +58,33 @@ export const PendingEntities = () => {
   const columns: TableColumn[] = [
     {
       title: <Typography>entityRef</Typography>,
+      sorting: true,
+      field: 'entity_ref',
+      customFilterAndSearch: (query, row: any) =>
+        row.entity_ref
+          .toLocaleUpperCase('en-US')
+          .includes(query.toLocaleUpperCase('en-US')),
```

Comment:
does it make sense to make the other fields case insensitive too?

i actually thought it was case insensitive by default 🤔 

(comment applies to the other file as well)